### ISSUE TITLE
Is it able to translate header title?

### ISSUE THREAD

**Issue by reevelau** [ssuperczynski/ngx-easy-table#224]
I am able to specify the header `title` in title property of `Columns`. 

I have tried to update the value (Columns.title) property on every language change event from `ngx-translate` module. But the column header text would only updated when user's mouse hover on ngx-easy-table.

I would like to know is there any proper way to do translate?

**Comment by ssuperczynski** [comment]
`columns` property is immutable so to apply changes you need to do:

```ts
translation()() {
  // some operation to apply translations
  this.columns = [...this.columns];
}
```

Could you check if this way works?

**Comment by reevelau** [comment]
Thanks, I did exactly the same way to translate the header fields. 

I was thinking if there is a template method for me to do it better, since the parent owned TranslateService is usable, which i dont understand how it work now....

**Comment by ssuperczynski** [comment]
Header template should be available in version 10.0. I've already stared to work on it. Similar issue #209
